src/assets/scss/main.scss

```diff
@@ -1,6 +1,14 @@
-@import "./abstracts/variables";
-@import "./layouts/header";
-@import "./layouts/footer";
-@import "./components/home";
-@import "./pages/home";
-@import "./pages/login";
+@import './abstracts/variables';
```

Comment:
Rename this folder `abstracts` to `utils`. Prefer using this SASS file structure: [7-1 pattern structure](https://gist.github.com/rveitch/84cea9650092119527bc)

Reply:
Thanks, I have corrected...